src/probnum/filtsmooth/statespace/sde.py

```diff
@@ -29,6 +29,7 @@ def __init__(
         self.dispmatfun = dispmatfun
         self.jacobfun = jacobfun
         self.precon = None
```

Comment:
`self.precon = None` is already set in `Transition.__init__()`. I suggest removing this line and moving the `super().__init__()` call to the top of `SDE`'s initializer.